compiler/semstmts.nim

```diff
@@ -251,7 +253,8 @@ proc semTry(c: PContext, n: PNode; flags: TExprFlags; expectedType: PType = nil)
       elif a.len == 1:
         # count number of ``except: body`` blocks
         inc catchAllExcepts
-
+        message(c.config, a.info, warnBareExcept,
+                  "The bare except clause is deprecated; use `except CatchableError, Defect:` instead")
```

Comment:
```suggestion
                  "The bare except clause is deprecated; use `except CatchableError:` instead")
```

we should not recommend catching Defect - this contradicts other sections of the manual

Reply:
Though many pre-existing examples still catch exceptions deriving from `Defect`.